v0/destinations/facebook_pixel/transform.js

```diff
@@ -0,0 +1,564 @@
+/* eslint-disable no-param-reassign */
+const sha256 = require("sha256");
+const get = require("get-value");
+const { CONFIG_CATEGORIES, MAPPING_CONFIG } = require("./config");
+const { EventType } = require("../../../constants");
+
+const {
+  constructPayload,
+  defaultPostRequestConfig,
+  defaultRequestConfig,
+  flattenJson,
+  isObject
+} = require("../../util");
+
+/**  format revenue according to fb standards with max two decimal places.
+ * @param revenue
+ * @return number
+ */
+
+const formatRevenue = revenue => {
+  return Number((revenue || 0).toFixed(2));
```

Comment:
```javascript
Number(("5" || 0).toFixed(2))
// Uncaught TypeError: ("5" || 0).toFixed is not a function
```

Which dangerous because based on datatype some number types will be coerced to string representation over the wire 

```ruby
irb(main):017:0> BigDecimal(5).as_json
=> "5.0"
```